Change from /usr/share/nzbget/webui to /usr/share/webapps/nzbget

---
diff --git a/cmake/install.cmake b/cmake/install.cmake
index 91c48b31..8a33b61c 100644
--- a/cmake/install.cmake
+++ b/cmake/install.cmake
@@ -4,10 +4,10 @@ set(DOC_FILES
 )
 set(SHARE_DIR ${CMAKE_INSTALL_PREFIX}/share/${PACKAGE})
 set(CONF_FILE ${CMAKE_SOURCE_DIR}/nzbget.conf)
-set(WEBUI_DIR ${CMAKE_SOURCE_DIR}/webui)
+set(WEBUI_DIR ${CMAKE_SOURCE_DIR}/webui/)
 set(DOC_FILES_DEST ${SHARE_DIR}/doc)
 set(CONF_TEMPLATE_FILE_DEST ${SHARE_DIR})
-set(WEBUI_DIR_DEST ${SHARE_DIR})
+set(WEBUI_DIR_DEST ${CMAKE_INSTALL_PREFIX}/share/webapps/${PACKAGE})
 set(BIN_FILE_DEST ${CMAKE_INSTALL_PREFIX}/bin)

 install(TARGETS ${PACKAGE} PERMISSIONS
@@ -23,7 +23,7 @@ install(FILES ${DOC_FILES} DESTINATION ${DOC_FILES_DEST})
 install(DIRECTORY ${WEBUI_DIR} DESTINATION ${WEBUI_DIR_DEST})

 file(READ ${CONF_FILE} CONFIG_CONTENT)
-string(REPLACE "WebDir=" "WebDir=${WEBUI_DIR_DEST}/webui" MODIFIED_CONFIG_CONTENT "${CONFIG_CONTENT}")
+string(REPLACE "WebDir=" "WebDir=${WEBUI_DIR_DEST}" MODIFIED_CONFIG_CONTENT "${CONFIG_CONTENT}")
 string(REPLACE "ConfigTemplate=" "ConfigTemplate=${CONF_TEMPLATE_FILE_DEST}/nzbget.conf" MODIFIED_CONFIG_CONTENT "${MODIFIED_CONFIG_CONTENT}")
 file(WRITE ${CMAKE_BINARY_DIR}/nzbget.conf "${MODIFIED_CONFIG_CONTENT}")
